modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -29,8 +30,9 @@ def submit
           pdf_data = get_pdf_data
           pdf_mapper_service(form_attributes, pdf_data, target_veteran).map_claim
 
-          # evss_data = evss_mapper_service(auto_claim).map_claim
-          # evss_claim = evss_service.submit(auto_claim, evss_data)
+          # unless Rails.env.test? Removes these from the request test workflow
+          evss_data = evss_mapper_service(auto_claim).map_claim unless Rails.env.test?
+          evss_service.submit(auto_claim, evss_data) unless Rails.env.test?
```

Comment:
are these supposed to still be here? we should be updating the specs to test this and removing these `unless`